Assets/MixedRealityToolkit.Examples/Demos/StandardShader/New feature/MaterialParameterCecker.cs

```diff
@@ -0,0 +1,374 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.UI;
+using Microsoft.MixedReality.Toolkit.UI;
+
+
+public class MaterialParameterCecker : MonoBehaviour
+{
+    //TargetMaterial
+    public Material material;
+    //SliderValueText
+    public Text  SliderValueText;
+    public List<Text> SliderValueTextList;
+
+    //HoverColor
+    private float hoverRed;
+    private float hoverGreen;
+    private float hoverBule;
+    
+    //InnerGlowColor
+    private float InnerGlowRed;
+    private float InnerGlowGreen;
+    private float InnerGlowBlue;
+
+    //EnvironmentColor x
+    private float EnvironmentXRed;
+    private float EnvironmentXGreen;
+    private float EnvironmentXBlue;
+
+    //EnvironmentColor y
+    private float EnvironmentYRed;
+    private float EnvironmentYGreen;
+    private float EnvironmentYBlue;
+
+    //EnvironmentColor z
+    private float EnvironmentZRed;
+    private float EnvironmentZGreen;
+    private float EnvironmentZBlue;
+
+
+    // RimLightColor
+    private float RimcolorRed;
+    private float RimcolorGreen;
+    private float RimcolorBlue;
+
+    private void SetText(float fvalue)
+    {
+        
+                SliderValueText.text = fvalue.ToString();
+        Debug.Log(SliderValueText.text);
+        Debug.Log(fvalue);
+    }
+
+    private void SetColorText(List<Text> textList,float xRvalue,float xGvalue, float xBvalue,float yRvalue, float yGvalue, float yBvalue,float zRvalue, float zGvalue, float zBvalue)
+    {
+        if (textList.Count >= 3)
```

Comment:
I think this should be: if (textList.Count >= 8) 
else you might access an element out of the list bounds